DiffPriv/__init__.py

```diff
@@ -24,9 +31,13 @@
 
 # Metadata
 __version__ = '0.1.1'

```

Comment:
Variables prefixed with an underscore will not be added to documentation by pdoc

Reply:
I know, this isn't for documentation on pdoc.

Reply:
Then use a `#` not a `"""` multi-line comment

Reply:
It will show up with `\_\_doc\_\_` however, which is why it's used.